packages/account-v2/src/containers/PersonalDetailsForm/PersonalDetailsForm.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { Fragment } from 'react';
+import { Form, Formik } from 'formik';
+import {
+    useActiveTradingAccount,
+    useGetAccountStatus,
+    useIsEuRegion,
+    useResidenceList,
+    useSettings,
+    useStatesList,
+} from '@deriv/api-v2';
+import { Checkbox, Text } from '@deriv-com/ui';
+import { FormSubHeader } from '../../components/FormSubHeader';
+import { useCurrentLandingCompany } from '../../hooks/useCurrentLandingCompany';
+import { AddressFields } from '../../modules/AddressFields';
+import { PersonalDetails } from './PersonalDetails';
+import { SupportProfessionalClient } from './SupportProfessionalClient';
+import { TaxInformation } from './TaxInformation';
+import { getPersonalDetailsInitialValues } from './utils';
+import { getPersonalDetailsValidationSchema } from '../../utils/personalDetailsUtils';
+
+export const PersonalDetailsForm = () => {
+    const { data: activeAccount } = useActiveTradingAccount();
```

Comment:
**NIT:** Just a thought, can we create one hook that wraps  all the below hooks and generate one response object that gives all the data of the hooks involved as well as computed values for `isSocialSignup`, `isVirtual` & `isSupportProfessionalClient` ? This way u can handle one single loader (which is missing) and error for PersonalDetails Form.